### PR TITLE
add ability to ignore projects on the org level

### PR DESCRIPTION
**Pull request by fat**

### CODE REVIEW

**Comment by dpup** [comment]
Can you remove DS_Store from this commit.  Then I'll merge.

File: extension/github.js [hunk 1]
```diff
@@ -250,9 +249,25 @@ function parseResponse(xhr) {
 function filterIgnored(list) {
   var filter = fetch('ignore_repos')
   if (!filter) return list
-  var ignoreList = filter.split(',').map(function (repo) { return repo.trim() })
+
+  var ignoreList = filter.split(',').map(function (repo) {
+    var matcher = repo.trim()
+
+    matcher = /* org/foo */ (/\/[^*]+/.test(matcher)) ? escapeRegExp(matcher) :
```
Comment (dpup) [inline extension/github.js:256]: Another option would be to construct a regexp that matched all these cases, instead of having a list of regexps.

Comment (dpup) [inline extension/github.js:256]: Move this comment up 3 lines.

**Comment by fat** [comment]
you had a ds_store and this commit removes it for you

**Comment by fat** [comment]
i can add it back if you want it i suppose

**Comment by dpup** [comment]
No, sorry, this is good.  My bad :P

**Comment by dpup** [comment]
This has been pushed to the chrome app store.

**Pull request merged**